apps/govquests-api/rails_app/db/seeds.rb

```diff
@@ -15,31 +15,51 @@ def create_action(action_data)
 read_document = [
   {
     action_type: "read_document",
-    display_data: {content: "Code of conduct"},
+    display_data: {
+      content: {
+        title: "Code of conduct",
+        description: ""
+      }
+    },
     action_data: {
       action_type: "read_document",
       document_url: "https://gov.optimism.io/t/code-of-conduct/5751"
     }
   },
   {
     action_type: "read_document",
-    display_data: {content: "Optimistic Vision"},
+    display_data: {
+      content: {
+        title: "Optimistic Vision",
+        description: ""
+      }
+    },
     action_data: {
       action_type: "read_document",
       document_url: "https://www.optimism.io/vision"
     }
   },
   {
     action_type: "read_document",
-    display_data: {content: "Working Constitution"},
+    display_data: {
+      content: {
+        title: "Working Constitution",
+        description: ""
+      }
```

Comment:
🤔 do we plan to have other keys in this dict? If not, this could perhaps simply have `title` and `description` directly instead of them being nested under `content`. wdyt?

Reply:
You're right, considering that it's already wrapped by a display_data key, I can't see a good reason to have the content: {}.

Reply:
@ribeirojose already changed on my current branch, It will be fixed on next PR